python/_tskitmodule.c

```diff
@@ -13254,9 +13254,11 @@ LsHmm_forward_matrix(LsHmm *self, PyObject *args)
             PyExc_ValueError, "haplotype array must have dimension (num_sites,)");
         goto out;
     }
-    err = tsk_ls_hmm_forward(self->ls_hmm, PyArray_DATA(haplotype_array),
-        compressed_matrix->compressed_matrix, TSK_NO_INIT);
-    if (err != 0) {
+    Py_BEGIN_ALLOW_THREADS err
+        = tsk_ls_hmm_forward(self->ls_hmm, PyArray_DATA(haplotype_array),
+            compressed_matrix->compressed_matrix, TSK_NO_INIT);
+    Py_END_ALLOW_THREADS if (err != 0)
+    {
```

Comment:
Did the linter put it all on one line? I'd expect this:
```suggestion
    Py_BEGIN_ALLOW_THREADS
    err = tsk_ls_hmm_forward(self->ls_hmm, PyArray_DATA(haplotype_array),
            compressed_matrix->compressed_matrix, TSK_NO_INIT);
    Py_END_ALLOW_THREADS
    if (err != 0)
    {
```

Reply:
I tried committing the suggested format, but it reverts to the older format.

Reply:
If you add a `;` after the macro then it will probably let you!

Reply:
Works! Thanks!